charts/rancher-istio/1.5.920/Chart.yaml

```diff
@@ -15,3 +15,4 @@ sources:
 - http://github.com/istio/istio
 tillerVersion: '>=2.7.2-0'
 version: 1.5.920
+kubeVersion: '< 1.21.0-0'
```

Comment:
same comment as above,  is it possible that someone could be using Istio v1 in an imported 1.21+ cluster today?